<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>machine</class>
 <widget class="QDialog" name="machine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>720</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;gestion des clients&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
  background: #6568F3;
  color: white;
  padding: 10px 12px;
border-radius:0px;
}

QTabBar::tab:selected {
    color: white;
   background-color:#FF7244;

}
QTabBar::tab:hover {
  background: #FF7244;
 }


QTabWidget::pane {
  top:0px; 
  background: rgb(245, 245, 245);
border-radius:10px;
} </string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des clients</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestion des commandes</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Gestion des produits</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gestion des machines</string>
    </attribute>
    <widget class="QGroupBox" name="gClientBg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1340</width>
       <height>720</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#gClientBg{
background-image:url(C:/Users/ALI/Desktop/bg);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>331</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  white;
border-radius:10px;

</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="Nom_f">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>128</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#Nom_f{
background-color: #F9FDFF;
border:1px solid rgba(222, 226, 230, 0.8);
padding-left:5px;
border-radius:5px;
color:#585856
}
#Nom_f:focus{
border:1px solid #6568f3;
outline:none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color:#341763;
color:#4a5bcf;
font-size:18px;</string>
       </property>
       <property name="text">
        <string>Formulaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>60</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="etat_f">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>128</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#etat_f{
background-color: #F9FDFF;
border:1px solid rgba(222, 226, 230, 0.8);
padding-left:5px;
border-radius:5px;
color:#585856;
}
#etat_f:focus{
border:1px solid #6568f3;
outline:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>247</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_f">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>128</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#id_f{
background-color: #F9FDFF;
border:1px solid rgba(222, 226, 230, 0.8);
padding-left:5px;
border-radius:5px;
color:#585856;
}
#id_f:focus{
border:1px solid #6568f3;
outline:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix_f">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>128</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#prix_f{
background-color: #F9FDFF;
border:1px solid rgba(222, 226, 230, 0.8);
padding-left:5px;
border-radius:5px;
color:#585856;
}
#prix_f:focus{
border:1px solid #6568f3;
outline:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red;</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_machine">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>265</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#ajouter_machine{
color:white;
background-color:#6568F3;
border-radius:5px;
}

#ajouter_machine:hover{
background-color:#4144f1;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_c">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>265</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#modifier_c{
color:white;
background-color:#03BF8A;
border-radius:5px;
}

#modifier_c:hover{
background-color:#02976d;
}
</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>245</y>
         <width>296</width>
         <height>1</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(222, 226, 230, 0.8);
border-width: 1px;
border: 2px solid rgba(222, 226, 230, 0.8);
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>27</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#6568F3;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>Liste des clients</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D6DBFD;
color:#4A5BCF;
padding:6px;
border-top-left-radius:10px;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>60</y>
        <width>221</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D6DBFD;
color:#4A5BCF;
padding:6px;
border-top-right-radius:10px;</string>
      </property>
      <property name="text">
       <string>     #</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>20</y>
        <width>201</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:12px;
padding:4px;
padding-left:10px;
padding-right:15px;
color:#7375f3;
font-size:16px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string extracomment="Recherche">Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_client">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>20</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#search_client{
background-color:white;
border-radius:12px;
}

#search_client:hover{
background-color:#D6DBFD;

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Client_resources.qrc">
        <normaloff>:/Resources/client_img/search.png</normaloff>:/Resources/client_img/search.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="trier_client_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>300</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#trier_client_2{
color:white;
background-color:#FF7244;
border-radius:5px;
}

#trier_client_2:hover{
background-color:#ff531a;
}</string>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trier_client">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>292</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#trier_client{
background-color:transparent;
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../sort.png</normaloff>../sort.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>302</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#6568F3;
font-size:16px;</string>
      </property>
      <property name="text">
       <string> Trier par</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>300</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:#585856;</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>Date d'ajout dans le système</string>
      </property>
      <item>
       <property name="text">
        <string>Date d'ajout dans le système</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>37</x>
        <y>70</y>
        <width>297</width>
        <height>1</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(222, 226, 230, 0.8);
border-width: 1px;
border: 2px solid rgba(222, 226, 230, 0.8);
</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf_b">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>14</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pdf_b{
background-color:transparent;
border-radius:15px;
image: url(:/Resources/client_img/pdf.png);
}

#pdf_b:hover{
background-color:#FF7244;

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../pdf.png</normaloff>../pdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QFrame" name="horizontalFrame">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>355</y>
        <width>601</width>
        <height>300</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>310</width>
        <height>310</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:none;</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>881</width>
        <height>188</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;border:none;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>864</width>
         <height>188</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="refreshBtn">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>20</y>
        <width>75</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#refreshBtn{
background-color:transparent;
image: url(:/Resources/client_img/refresh.png);

}

#refreshBtn:hover{
	image: url(:/Resources/client_img/refresh-button.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D6DBFD;
color:#4A5BCF;
padding:6px;</string>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>60</y>
        <width>101</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D6DBFD;
color:#4A5BCF;
padding:6px;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>60</y>
        <width>181</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D6DBFD;
color:#4A5BCF;
padding:6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>60</y>
        <width>281</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D6DBFD;
color:#4A5BCF;
padding:6px;</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Client_resources.qrc"/>
 </resources>
 <connections/>
</ui>
